app/assets/javascripts/live_search.js

```diff
@@ -65,10 +66,26 @@
             GOVUK.analytics.trackPageview(newPath);
           }
         }.bind(this)
-      );
+      )
     }
   };
 
+  LiveSearch.prototype.fireTextAnalyticsEvent = function(event) {
+    if (GOVUK.analytics && GOVUK.analytics.trackPageview) {
```

Comment:
This check (`GOVUK.analytics && GOVUK.analytics.trackPageview`) is done elsewhere in the file. Do you think it would be worth adding a new function for `LiveSearch.canTrackPageview`?